framework/ui/internal/uiconfiguration.cpp

```diff
@@ -63,7 +63,8 @@ async::Channel<int> UiConfiguration::fontSizeChanged()
 
 float UiConfiguration::guiScaling() const
 {
-    return static_cast<float>(Ms::guiScaling);
+    //! TODO
```

Comment:
Please, add a description for TODO. What should this method do?

Reply:
added